docs/Users_Guide/load_data.rst

```diff
@@ -359,3 +721,16 @@ Troubleshooting
 
   * - Solution:
     - This error is caused when attempting to load data into a database that does not exist.  You will need to create the database, set up the appropriate privileges as outlined above, and load the schema using the mv_mysql.sql file.
+
+  * -  Error:
+    -  /full-path-to/xyz.xml is not valid and may contain a recursive payload or an excessively large payload
+
+  * - Solution:
+    - This error is typically encountered when one of the following conditions exist as a result of failing the XML validation step:
+
+         - the order of the elements in your XML specification file is inconsistent with the order expected
```

Comment:
```suggestion
         - the order of the elements in the XML specification file is inconsistent with the order expected
```
We try to keep "you" language out of the guides.  Suggest changing to "the" unless you think it is unclear.